docs/getting_oss_rules_of_play_rating.md

```diff
@@ -113,28 +118,30 @@ If everything is correctly set up, the output is going to look like the followin
 [+]   Value:........9.0  out of 10.0
 [+]   Confidence:...Max (10.0 out of 10.0)
 [+]   Based on:...19 features
-[+]       Are vulnerability alerts enabled?......................Yes
-[+]       Does README mention REUSE?.............................Yes
-[+]       Does it have LICENSES directory with licenses?.........Yes
-[+]       Does it have a README file?............................Yes
-[+]       Does it have a contributing guideline?.................Yes
-[+]       Does it have a license file?...........................Yes
-[+]       Does it have a security policy?........................Yes
-[+]       Does it have a team with push privileges on GitHub?....Yes
-[+]       Does it have an admin team on GitHub?..................Yes
-[+]       Does it have enough admins on GitHub?..................Yes
-[+]       Does it have enough teams on GitHub?...................Yes
-[+]       Does it have unresolved vulnerability alerts?..........No
-[+]       Does it use an allowed license?........................Yes
-[+]       Does teams have enough members on GitHub?..............Yes
-[+]       Does the contributing guideline have required text?....Yes
-[+]       Does the license have disallowed content?..............No
-[+]       If a project's README doesn't contain required info:...Yes
-[+]       Is it compliant with REUSE rules?......................Yes
-[+]       Is it registered in REUSE?.............................Yes
-[+]   Explanation:..Found 1 recommendations
+[+]       Are vulnerability alerts enabled?.........................Yes
+[+]       Does README mention REUSE?................................Yes
+[+]       Does it have LICENSES directory with licenses?............Yes
+[+]       Does it have a README file?...............................Yes
+[+]       Does it have a code of conduct guideline?.................No
+[+]       Does it have a contributing guideline?....................Yes
+[+]       Does it have a license file?..............................Yes
+[+]       Does it have a security policy?...........................Yes
+[+]       Does it have a team with push privileges on GitHub?.......Yes
+[+]       Does it have an admin team on GitHub?.....................Yes
+[+]       Does it have enough admins on GitHub?.....................Yes
+[+]       Does it have enough teams on GitHub?......................Yes
+[+]       Does it have unresolved vulnerability alerts?.............No
+[+]       Does it use an allowed license?...........................No
```

Comment:
Shouldn't this be Yes, if this is the same project?

Reply:
Not sure why the allowed license part is saying no, i tried the same with openui5 repository and for that it also showed No for the ´Does it use an allowed license?´, 
So, I cloned the current master branch and executed the code for openui5 and code-pal-for-abap and for both it showed No for the ´Does it use an allowed license?´
maybe this is an issue on master which is causing this error, as I have only implemented the Code of conduct thing and did not touch any other part of code.

Reply:
@ajinkyapatil8190 this is the final part. Please check if you can find a suitable project to generate a report with Rating value > 0